src/spryker/application/skeleton/composer-harness.json.twig

```diff
@@ -67,8 +85,8 @@
     "test-quality": [
       "@generate",
       "@phpstan",
-      "APPLICATION_ENV=development vendor/bin/console code:sniff:style",
-      "APPLICATION_ENV=development vendor/bin/console code:sniff:architecture"
+      "find src/ -maxdepth 1 -mindepth 1 -path 'src/Pyz' -prune -o  -print | xargs -n1 env APPLICATION_ENV=development DEVELOPMENT_CONSOLE_COMMANDS=1 vendor/bin/console code:sniff:style",
+      "APPLICATION_ENV=development DEVELOPMENT_CONSOLE_COMMANDS=1 vendor/bin/console code:sniff:architecture"
```

Comment:
DEVELOPMENT_CONSOLE_COMMANDS=1   shouldn't be needed as the variable should be service environment

Reply:
We call composer test-quality on the static build so I was following the logic of APPLICATION_ENV=development being set here.